pywbem_mock/_cimrepository.py

```diff
@@ -828,49 +855,51 @@ def EnumerateClasses(self, **params):
             CIMError: CIM_ERR_INVALID_CLASS: class defined by the classname
             parameter does not exist.
         """
-        namespace = self.namespace
+        # namespace = self.namespace
```

Comment:
Since the overwriting of the input namespace was not detected by the test cases, it seems that additional tests are needed for the combinations of provided namespace and default namespace.

Reply:
Withdrawing this comment, see previous comment.

Reply:
Agreed, the test cases are incomplete.  Also, I really need a unit test for CIMRepository itself.

Reply:
the MainProvider variable self.namespace has been completely removed.